src/renderer/components/+helm-releases/release-details/release-details-model/release-details-model.injectable.tsx

```diff
@@ -0,0 +1,312 @@
+        <p>
+          Release
+          {" "}
+          <b>{name}</b>
+          {" successfully updated!"}
+        </p>,
+      );
+
+      await this.loadConfiguration();
+    },
+  };
+
+  readonly onlyUserSuppliedValuesAreShown: OnlyUserSuppliedValuesAreShownToggle = {
+    value: observable.box(false),
+
+    toggle: action(async () => {
+      const value = this.onlyUserSuppliedValuesAreShown.value;
+
+      value.set(!value.get());
+
+      await this.loadConfiguration();
+    }),
+  };
+
+  load = async () => {
+    runInAction(() => {
+      this.isLoading.set(true);
+    });
+
+    const { name, namespace } = this.dependencies.targetRelease;
+
+    const detailedRelease = await this.dependencies.callForHelmRelease(
+      name,
+      namespace,
+    );
+
+    runInAction(() => {
+      this.detailedRelease.set(detailedRelease);
+    });
+
+    await this.loadConfiguration();
+
+    runInAction(() => {
+      this.isLoading.set(false);
+    });
+  };
+
+  private loadConfiguration = async () => {
+    runInAction(() => {
+      this.configuration.isLoading.set(true);
+    });
+
+    const { name, namespace } = this.release;
+
+    const configuration =
+      await this.dependencies.callForHelmReleaseConfiguration(
+        name,
+        namespace,
+        !this.onlyUserSuppliedValuesAreShown.value.get(),
+      );
+
+    runInAction(() => {
+      this.configuration.isLoading.set(false);
+      this.configuration.nonSavedValue.set(configuration);
+    });
+  };
+
+  @computed get release() {
+    const detailedRelease = this.detailedRelease.get();
+
+    assert(detailedRelease, "Tried to access release before load");
+
+    return toHelmRelease(detailedRelease.release);
+  }
+
+  @computed private get details() {
+    const detailedRelease = this.detailedRelease.get();
+
+    assert(detailedRelease, "Tried to access details before load");
+
+    return detailedRelease.details;
+  }
+
+  @computed get notes() {
+    return this.details.info.notes;
+  }
+
+  @computed get groupedResources(): MinimalResourceGroup[] {
+    return pipeline(
+      this.details.resources,
+      groupBy((resource) => resource.kind),
+      (grouped) => Object.entries(grouped),
+
+      map(([kind, resources]) => ({
+        kind,
+
+        resources: resources.map(
+          toMinimalResourceFor(this.dependencies.getResourceDetailsUrl, kind),
+        ),
+
+        isNamespaced: resources.some(
+          (resource) => !!resource.metadata.namespace,
+        ),
+      })),
+    );
+  }
+
+  @computed get activeTheme() {
+    return this.dependencies.themeStore.activeTheme.type;
+  }
+
+  close = () => {
+    this.dependencies.navigateToHelmReleases();
+  };
+
+  startUpgradeProcess = () => {
+    this.dependencies.createUpgradeChartTab(this.release);
+
+    this.close();
+  };
+
+  @computed get id() {
+    const { name, namespace } = this.dependencies.targetRelease;
+
+    return `${namespace}/${name}`;
+  }
```

Comment:
nit: this seems strange, why is not just assigned once at construction time?

Reply:
Sure, changed.